src/IntegrationUtils.ts

```diff
@@ -28,54 +28,100 @@ const basicIntegration : string = 'Basic - Apache Camel K Integration (no Config
 const configMapIntegration : string = 'ConfigMap - Apache Camel K Integration with Kubernetes ConfigMap';
 const secretIntegration : string = 'Secret - Apache Camel K Integration with Kubernetes Secret';
 
+const choiceList = [ basicIntegration, 
+	configMapIntegration, 
+	secretIntegration ];
+
 export async function startIntegration(context: vscode.Uri): Promise<any> {
 	return new Promise <any> ( async (resolve, reject) => {
-		await vscode.window.showQuickPick([ basicIntegration, configMapIntegration, secretIntegration ],
-		{
+		const choice : unknown = await vscode.window.showQuickPick(choiceList, {
```

Comment:
shouldn't we use "string | undefined" instead of "unknown"?

Reply:
Considering I had no clue I could do that. :) I just tried it and it worked great. Changing now.